docs/design/virtualization.md

```diff
@@ -40,7 +40,7 @@ Kata Containers with QEMU has complete compatibility with Kubernetes.
 
 Depending on the host architecture, Kata Containers supports various machine types,
 for example `pc` and `q35` on x86 systems, `virt` on ARM systems and `pseries` on IBM Power systems. The default Kata Containers
-machine type is `pc`. The machine type and its [`Machine accelerators`](#machine-accelerators) can
+machine type is `q35`. The machine type and its [`Machine accelerators`](#machine-accelerators) can
```

Comment:
Thanks, but the original text is still rather unclear as the default machine type is _only_ `q35` on `x86_64`. I'd consider reworking this paragraph by:

- Changing the machine type text into a table like this:

  | Architecture | Machine type |
  |-|-|
  | `arm64` (`aarch64`) | `virt` |
  | `ppc64le` | `pseries` |
  | `s390x` | `s390-ccw-virtio` |
  | `x86_64` | `q35` |

- Adding:

  > See https://github.com/kata-containers/kata-containers/tree/main/src/runtime/arch for further details.

  ... since those architecture-specific configuration files set `MACHINETYPE := ...`, and it's those values that end up in `configuration-qemu.toml`.

Reply:
I just saw this small mistake and wanted to make a quick PR to fix it, without intention to put much effort into it.